### ISSUE TITLE
TF2 no longer works on Fedora (latest version)

### ISSUE THREAD

**Issue by n-stepan0v** [ValveSoftware/Source-1-Games#3647]
[tf22.txt](https://github.com/ValveSoftware/Source-1-Games/files/7034444/tf22.txt)
[tf2.txt](https://github.com/ValveSoftware/Source-1-Games/files/7034447/tf2.txt)
```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM)2 Duo CPU     E7500  @ 2.93GHz
    CPU Family:  0x6
    CPU Model:  0x17
    CPU Stepping:  0xa
    CPU Type:  0x0
    Speed:  2936 Mhz
    2 logical processors
    2 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Unsupported
    AES:  Unsupported
    AVX:  Unsupported
    AVX2:  Unsupported
    AVX512F:  Unsupported
    AVX512PF:  Unsupported
    AVX512ER:  Unsupported
    AVX512CD:  Unsupported
    AVX512VNNI:  Unsupported
    SHA:  Unsupported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Linux 5.x (64 bit)
    Kernel Name:  Linux
    Kernel Version:  5.13.12-xm1.0.fc34.x86_64
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12011000
    X Window Manager:  Xfwm4
    Steam Runtime Version:  steam-runtime_0.20210630.0

Video Card:
    Driver:  NVIDIA Corporation GeForce GTS 450/PCIe/SSE2
    Driver Version:  4.6.0 NVIDIA 390.144
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x10de
    DeviceID:  0xdc4
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1280 x 1024
    Desktop Resolution: 1280 x 1024
    Primary Display Size: 13.31" x 10.63" (17.01" diag)
                                            33.8cm x 27.0cm (43.2cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 1024 MB
    Supported MSAA Modes:  2x 4x 8x 16x 

Sound card:
    Audio device: Realtek ALC888

Memory:
    RAM:  3928 Mb

VR Hardware:
    VR Headset: None detected

Miscellaneous:
    UI Language:  English
    LANG:  ru_RU.utf8
    Total Hard Disk Space Available:  61416 Mb
    Largest Free Hard Disk Block:  12892 Mb

Storage:
    Number of SSDs: 0
    SSD sizes: %s1
    Number of HDDs: 0
    HDD sizes: %s1

The runtime information tool is preparing a report, please wait...
```

**Comment by n-stepan0v** [comment]
```
Computer Information:
    Manufacturer:  Unknown
    Model:  Unknown
    Form Factor: Desktop
    No Touch Input Detected

Processor Information:
    CPU Vendor:  GenuineIntel
    CPU Brand:  Intel(R) Core(TM)2 Duo CPU     E7500  @ 2.93GHz
    CPU Family:  0x6
    CPU Model:  0x17
    CPU Stepping:  0xa
    CPU Type:  0x0
    Speed:  2936 Mhz
    2 logical processors
    2 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Unsupported
    AES:  Unsupported
    AVX:  Unsupported
    AVX2:  Unsupported
    AVX512F:  Unsupported
    AVX512PF:  Unsupported
    AVX512ER:  Unsupported
    AVX512CD:  Unsupported
    AVX512VNNI:  Unsupported
    SHA:  Unsupported
    CMPXCHG16B:  Supported
    LAHF/SAHF:  Supported
    PrefetchW:  Unsupported

Operating System Version:
    Linux 5.x (64 bit)
    Kernel Name:  Linux
    Kernel Version:  5.13.12-xm1.0.fc34.x86_64
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12011000
    X Window Manager:  Xfwm4
    Steam Runtime Version:  steam-runtime_0.20210630.0

Video Card:
    Driver:  NVIDIA Corporation GeForce GTS 450/PCIe/SSE2
    Driver Version:  4.6.0 NVIDIA 390.144
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x10de
    DeviceID:  0xdc4
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1280 x 1024
    Desktop Resolution: 1280 x 1024
    Primary Display Size: 13.31" x 10.63" (17.01" diag)
                                            33.8cm x 27.0cm (43.2cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 1024 MB
    Supported MSAA Modes:  2x 4x 8x 16x 

Sound card:
    Audio device: Realtek ALC888

Memory:
    RAM:  3928 Mb

VR Hardware:
    VR Headset: None detected

Miscellaneous:
    UI Language:  English
    LANG:  ru_RU.utf8
    Total Hard Disk Space Available:  61416 Mb
    Largest Free Hard Disk Block:  12892 Mb

Storage:
    Number of SSDs: 0
    SSD sizes: %s1
    Number of HDDs: 0
    HDD sizes: %s1

The runtime information tool is preparing a report, please wait...
```

**Comment by kisak-valve** [comment]
Hello @n-stepan0v, if you're still using this hardware, unfortunately, you'll now be blocked by https://github.com/ValveSoftware/steam-runtime/issues/420 now that Team Fortress 2 is run inside the Steam Linux Runtime - Sniper container environment.

Closing in favor of the steam-runtime issue report.
